ikarus/linearalgebra/dirichletvalues.hh

```diff
@@ -72,6 +72,13 @@ namespace Ikarus {
       f(basis_, dirichletFlagsBackend);
     }
 
+    /**
+     * \brief Function to fix (set boolean values to true or false) of degrees of freedom
+     *
+     * \param i An index indicating the DOF number to be fixed
+     */
+    void fixIthDOF(std::size_t i) { dirichletFlags[i] = true; }
```

Comment:
This function should receive `typename Basis::MultiIndex` as a type and then you fix the value in `dirichletFlagsBackend`.

This is much more portable

Reply:
Modified